### PR TITLE
add new tasks and fix prettierc to reflect coding style in the code

### PR DESCRIPTION
**Pull request by vmaark**
**What does this pull request do? Explain your changes. (required)**
- add some new tasks used in local dev
- fixed prettierc usage

### CODE REVIEW

**Review by kyriediculous — COMMENTED** [review]
Tasks are supposed to be like a CLI for the protocol to interact with it

These dummy tasks should be scripts instead.

File: .prettierrc [hunk 1]
```diff
@@ -1,12 +1,12 @@
-
 {
   "arrowParens": "avoid",
   "bracketSpacing": true,
-  "endOfLine":"auto",
+  "endOfLine": "auto",
   "printWidth": 120,
-  "singleQuote": false,
+  "singleQuote": true,
```
Comment (kyriediculous) [inline .prettierrc:6]: This can not work.

This is prettier for solidity files only and solidity does not support single quote strings

**Review by kyriediculous — CHANGES_REQUESTED** [review]
Can only be merged if above is addressed, but this is not on the critical path

File: tasks/add_liquidity.ts [hunk 2]
```diff
@@ -24,14 +24,17 @@ task('add-liquidity', 'adds liquidity to pool')
...
+      await Tenderizer.tenderToken()
+    )) as TenderToken
     const Steak: ERC20 = (await ethers.getContractAt('ERC20', await Tenderizer.steak())) as ERC20
     const TenderSwap: TenderSwap = (await ethers.getContractAt(
       'TenderSwap',
       await Tenderizer.tenderSwap()
     )) as TenderSwap
 
-    const deadline = Math.floor((new Date().getTime() + 20 * 60000) / 1000)
+    const deadline = Math.floor((new Date(1674139736000).getTime() + 20 * 60000) / 1000)
```
Comment (reubenr0d) [inline tasks/add_liquidity.ts:37]: Umm, I think you missed removing the hardcoded deadline here?

File: tasks/increase_timestamp.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { task, types } from 'hardhat/config'
```
Comment (reubenr0d) [inline tasks/increase_timestamp.ts:1]: Hmm, not sure if we should add this task to the repo, even the sending ETH one above, and set_timestamp

File: tasks/add_liquidity.ts (hunk 2 above)
Comment (vmaark) [reply tasks/add_liquidity.ts:37]: 👍

File: tasks/increase_timestamp.ts (hunk 3 above)
Comment (vmaark) [reply tasks/increase_timestamp.ts:1]: @kyriediculous already wrote that (some?) of these should be moved to scripts instead of tasks.

**Pull request closed without merging**